### PR TITLE
Dp

### PR DESCRIPTION
**Pull request by kirtisingh3008**
**Fixes issue:**

I have added Count all subsequences having product less than K  to dynamic problem folder in cpp language.
**Changes:**

### CODE REVIEW

File: code/dynamic_programming/src/Count_Subsequence_Having_Product_Less_Than_K/Solution.cpp [hunk 1]
```diff
@@ -0,0 +1,67 @@
+//Given a positive array, find the number of subsequences having product smaller than K.
```
Comment (walidbarakat) [inline code/dynamic_programming/src/Count_Subsequence_Having_Product_Less_Than_K/Solution.cpp:1]: not needed

File: code/dynamic_programming/src/Count_Subsequence_Having_Product_Less_Than_K/Solution.cpp [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+// LINK - https://www.geeksforgeeks.org/count-subsequences-product-less-k/#
+#include<bits/stdc++.h>
```
Comment (walidbarakat) [inline code/dynamic_programming/src/Count_Subsequence_Having_Product_Less_Than_K/Solution.cpp:3]: not a good practice to use such header, can you please use only the headers needed?

File: code/dynamic_programming/src/Count_Subsequence_Having_Product_Less_Than_K/Solution.cpp [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        a.push_back(x);
+    }
+
+    int k;
+    cout<<"Enter the value of k"<<endl;
+    cin>>k;
+
+    cout<<solve(a,n,k)<<endl;
+    return 0;
+}
```
Comment (walidbarakat) [inline code/dynamic_programming/src/Count_Subsequence_Having_Product_Less_Than_K/Solution.cpp:67]: let the last line be a blank one always

**Review by walidbarakat — CHANGES_REQUESTED** [review]
Hi, can you please consider the comments and as well please give a meaningful name to the PR

File: code/dynamic_programming/src/Count_Subsequence_Having_Product_Less_Than_K/Solution.cpp (hunk 3 above)
Comment (kirtisingh3008) [reply code/dynamic_programming/src/Count_Subsequence_Having_Product_Less_Than_K/Solution.cpp:67]: okay sure , sir will make the changes .

**Pull request merged**